### PR TITLE
hack: polish scripts

### PR DESCRIPTION
**Pull request by ahmetb**

### CODE REVIEW

**Comment by m-okeefe** [comment]
Small thing: when I first ran `./make-release-artifacts.sh`, I saw:

```
./hack/make-release-artifacts.sh: line 57: realpath: command not found
Successfully saved merged manifests to .
``` 

Ran `brew install coreutils`, then it was fine: 

```
Successfully saved merged manifests to /Users/mokeefe/ ....
```

Add note that you need `realpath`?

File: hack/make-docker-images.sh [hunk 1]
```diff
@@ -12,28 +14,28 @@
...
+while IFS= read -d $'\0' -r dir; do
+    # build image
+    svcname="$(basename "${dir}")"
+    image="${REPO_PREFIX}/$svcname:$TAG"
+    (
+        cd "${dir}"
+        log "Building: ${image}"
+        docker build -t "${image}" .
+
+        echo "Pushing: ${image}"
```
Comment (m-okeefe) [inline hack/make-docker-images.sh]: s/echo/log ?

File: hack/make-release-artifacts.sh [hunk 2]
```diff
@@ -14,42 +16,42 @@
...
+    while IFS= read -d $'\0' -r file; do
+        cat "${file}"
+        echo "---"
+    done < <(find "${src_manifest_dir}" -name '*.yaml' -type f -print0)
+}
 
-# remove empty lines 
-gsed -r -i '/^\s*$/d' $tmp 
+# read and merge all manifests
+out_manifest="$(read_manifests)"
```
Comment (m-okeefe) [inline hack/make-release-artifacts.sh]: trying to figure out how you're getting 1 google header to stay? 
also it's 2018 vs. 2019 now?

**Comment by ahmetb** [comment]
> Add note that you need `realpath`?

we can remove `realpath` dependency.

File: hack/make-release-artifacts.sh (hunk 2 above)
Comment (ahmetb) [reply hack/make-release-artifacts.sh]: I removed it because the license was there already, in the original files.

**Pull request merged**